### ISSUE TITLE
Temp no longer goes up to 260

### ISSUE THREAD

**Issue by janlukes** [BotInABoxER/marlin2-for-wanhao-i3#4]
I updated to this fw and my printer no longer can reach 260 on hotend and 110 on hotbed. Only 245 on hotend and 90 on hotbed is available. What confuses me is that configuration.h has 260 set as max temp. Did you set 245 as when you calibrated the thermistor?

**Comment by mtpiercey** [comment]
Can confirm. Yeah, that's pretty weird; I guess I never noticed it since I only print with PLA, and before I got an all-metal hotend I wouldn't even have dreamed of temps that high...

First off, I'd recommend trying the official version of Marlin. This is an outdated port that I ended up merging, so there should now be official specific configuration for the i3 v2.1 (which probably still needs to be tweaked for your situation).

I'm also thinking I should archive or delete this repo, since there are seem to be a lot of printer-specific issues I didn't run into/think about when I cooked up this config for my printer/usage, and I'm not really qualified to maintain an old fork of a project like this.

Finally, https://github.com/MarlinFirmware/Marlin/issues/17693 is likely your answer here (since thermal runway protection is enabled, the firmware won't let the extruder get hotter than max - 15 degrees, since at the max it shuts off the heaters in case of an emergency). I know that's kinda non-intuitive; I didn't realize it either until I checked.

So, I recommend:
  - Getting ahold of the official Marlin source/configuration
  - Editing the Wanhao/Duplicator i3 v2.1 config to allow the max temps to be 20/30 degrees higher than they are, or whatever floats your boat
  - Compiling the latest stable Marlin (2.0 branch)
  - Uploading it as a .hex to your printer
  - Profit?
